torch/distributions/utils.py

```diff
@@ -66,3 +66,21 @@ def broadcast_all(*values):
         for idx in scalar_idxs:
             values[idx] = torch.Tensor([values[idx]])
     return values
+
+
+def tril_mask(value):
```

Comment:
Doesn't `torch.tril(value.new().byte().resize_as_(value).fill_(1))` do the job?
`torch.tril(torch.ones_like(value).byte())` would work too, but would be slower.